crates/noirc_evaluator/src/errors.rs

```diff
@@ -44,8 +44,6 @@ pub enum InternalError {
     MissingArg { name: String, arg: String, location: Option<Location> },
     #[error("ICE: {name:?} should be a constant")]
     NotAConstant { name: String, location: Option<Location> },
-    #[error("{name:?} is not implemented yet")]
-    NotImplemented { name: String, location: Option<Location> },
```

Comment:
This wasn't being used anywhere but on the `RecursiveAggregation` case for black box funcs. Every instance of it being used also did not have location information which makes it less useful than `unimplemented!()`. For the time being I have removed this InternalError